src/utils/docs/user.yaml

```
@@ -0,0 +1,233 @@
+            country:
+              type: string
+            city:
+              type: string
+        photo:
+          type: string
+        professionalSummary:
+          type: string
+        mainSubjects:
+          type: array
+          items:
+            type: string
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)